content/1_General/Contributing.mdx

```diff
@@ -59,8 +59,11 @@ See the [Content Documentation](https://github.com/cpinitiative/usaco-guide/blob
 	- nice markdown editor
 	- supports Katex
 - [Sublime Text](https://www.sublimetext.com/)
-	- `.md` syntax highlighting is fine, is also ok for `.mdx`
-	- You can open a `.mdx` file and set syntax highlighting to be the same as `.md` with `View -> Syntax -> Open all with current extension as ... -> Markdown -> Markdown`.
+  - `.md` syntax highlighting is fine, is also ok for `.mdx`
+  - You can open a `.mdx` file and set syntax highlighting to be the same as `.md` with `View -> Syntax -> Open all with current extension as ... -> Markdown -> Markdown`.
+- [Obsidian](https://obsidian.md/)
+  - Markdown editor
+  - Automatically compiles $\LaTeX$ (doesn't require an installation)
```

Comment:
indentation? (preferably use tabs? though honestly I don't mind spaces for markdown lists as long as the file is consistent)

Reply:
I think all md lists consistently use 2 spaces